charts/stable/external-dns/Chart.yaml

```
@@ -36,5 +36,5 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/truecharts/charts/tree/master/charts/stable/external-dns
 type: application
-version: 6.8.0
+version: 6.8.1
```

```suggestion
version: 6.9.0
```
This is a feature addition, not a bug fix